.github/workflows/gh-pages.yml

```
@@ -0,0 +1,30 @@
+name: GitHub pages
```

Use verbs in workflows names 🙂 

```suggestion
name: Deploy to GitHub Pages
```